component/namespace-policies.jsonnet

```diff
@@ -131,23 +131,14 @@ local organizationNamespaces = kyverno.ClusterPolicy('organization-namespaces')
             value: '',
           },
         ],
-        context: [
-          {
-            name: 'groups',
-            apiCall: {
-              urlPath: '/apis/user.openshift.io/v1/groups',
-              jmesPath: 'items',
-            },
-          },
-        ],
         validate: {
           message: 'Creating namespace for {{request.object.metadata.labels."appuio.io/organization"}} but {{request.userInfo.username}} is not in organization',
           deny: {
             conditions: [
               {
-                key: '{{request.userInfo.username}}',
-                operator: 'NotIn',
-                value: "{{groups[?metadata.name=='{{request.object.metadata.labels.\"appuio.io/organization\"}}'].users[]}}",
+                key: '{{request.object.metadata.labels."appuio.io/organization"}}',
+                operator: 'In',
+                value: '{{request.userInfo.groups}}',
```

Comment:
Maybe I'm confused because Friday, but shouldn't we deny the request if the organization label is **not** in the list of groups?

Reply:
good catch, not sure what happened there